docs/source/getting-started/installation.rst

```diff
@@ -140,6 +140,21 @@ Next, get your `Account ID <https://developers.cloudflare.com/fundamentals/get-s
 
   Support for R2 is in beta. Please report and issues on `Github <https://github.com/skypilot-org/skypilot/issues>`_ or reach out to us on `Slack <http://slack.skypilot.co/>`_.
 
+IBM
```

Comment:
Could we move this above Cloudflare, since it's a compute cloud provider?

Reply:
How about "Lambda -> IBM -> Cloudflare" (in order of addition)? As more clouds are added we can figure out the best ordering in the future.